src/engine.c

```diff
@@ -34,13 +34,48 @@ char *exp_names[MAX_EXPANSION + 1] =
 	"The Brink of War",
 	"Alien Artifacts",
 	"Xeno Invasion",
+	"Rebel vs Imperium (without Gathering Storm)",
```

Comment:
With the catch that Rebel vs Imperium (only) is in fact Base Game + Rebel vs Imperium. But this is minor, I can accept RvI (only). The abbreviation I used is in fact RvIo, so this is consistent.